handlers/camera.py

```diff
@@ -0,0 +1,51 @@
+from flask import request
+from gphoto2.gphoto import GPhoto
+from handlers.main import returnResponse
+
+Cam = GPhoto()
```

Comment:
Avoiding the need to create a top-level object like this is why I did the whole `create_app` pattern.  Blueprints could let you have the best of both worlds.  It's probably not important in this case, but it definitely makes things easier to (re-)use in other ways if there isn't a forced object creation like this.  The complexity of setting it all up may not be worth it in this case.